roles/container-engine/docker/tasks/main.yml

```diff
@@ -85,32 +75,6 @@
   with_items: "{{ docker_repo_info.repos }}"
   when: not (ansible_os_family in ["Flatcar Container Linux by Kinvolk", "RedHat", "Suse", "ClearLinux"] or is_ostree) and (docker_repo_info.repos|length > 0)
 
-- name: ensure docker-engine repository public key is installed
-  action: "{{ dockerproject_repo_key_info.pkg_key }}"
-  args:
-    id: "{{ item }}"
-    url: "{{ dockerproject_repo_key_info.url }}"
-    state: present
-  register: keyserver_task_result
-  until: keyserver_task_result is succeeded
-  retries: 4
-  delay: "{{ retry_stagger | d(3) }}"
-  with_items: "{{ dockerproject_repo_key_info.repo_keys }}"
-  environment: "{{ proxy_env }}"
-  when:
-    - not (ansible_os_family in ["Flatcar Container Linux by Kinvolk", "RedHat", "Suse", "ClearLinux"] or is_ostree)
-    - use_docker_engine is defined and use_docker_engine
```

Comment:
Why can we remove this task?

Reply:
we have both `docker_repo_info` and `dockerproject_repo_info` pointing to the same base url / not the same directory
dockerproject was the old repo that was deprecated some times ago

Reply:
Also you can see at https://download.docker.com/linux/debian/dists/ that the dockerproject folders are not presents anymore
And CI pass ;)

Reply:
I see.
I found the commit 38688a44866d5ee612f80c16ef81e135d1347653 to show dockerproject.org was deprecated in 2017 and has gone down already.